integration-tests/bats/stats.bats

```diff
@@ -208,33 +208,13 @@ teardown() {
     cd repo2
 
     dolt sql -q "alter table xy add index y2 (y)"
-    dolt sql -q "insert into xy values (0,0), (1,0), (2,0), (3,0), (4,0), (5,0)"
+    dolt sql -q "insert into xy values (0,0), (1,0), (2,0), (3,0), (4,0), (5,0), (6,1), (7,2), (8,3), (9,4)"
 
-    # setting variables doesn't hang or error
-    dolt sql -q "SET @@persist.dolt_stats_auto_refresh_enabled = 1;"
```

Comment:
Why were these deleted?

Reply:
replaced with the analyze below, we already test background loading figured this was overkill